Kronos-Shifts-Connector/README.md

```diff
@@ -206,45 +264,93 @@ Once the ARM Template deployment is successful, one final operation is to ensure
 2.	Navigate to the resource group that was created at the time of ARM Template deployment
 3.	Navigate to the storage account that was created from the ARM Template deployment. The screen should resemble the figure below: 
 
-**Figure 7.** Storage account overview
+**Figure 17.** Storage account overview
 ![figure7](images/figure7.png)
 
 4.	Navigate into the containers, by clicking on the link that reads *Containers* from Figure 7 above
 5.	Upon navigation to the containers, the ARM Template should provision a blob container called “templates”, and the screen should resemble below:
 
-**Figure 8.** Templates blob container
+**Figure 19.** Templates blob container
 ![figure8](images/figure8.png)
 
 6. Navigate inside of the "templates" blob container, and the screen should resemble the next screenshot below: 
 
-**Figure 9.** Navigation inside of the "templates" blob.
+**Figure 19.** Navigation inside of the "templates" blob.
 ![figure9](images/figure9.png)
 
 User Creation through the Teams Admin Portal
 1.    Navigate to the [Microsoft Teams Admin Portal](https://admin.teams.microsoft.com)
 2.	Sign In with your AAD Tenant Admin credentials
 3.	You will be presented with the following view once the sign in is successful:
 
-**Figure 10.** Home page of the Teams Admin portal
+**Figure 20.** Home page of the Teams Admin portal
 ![figure10](images/figure10.png)
 
 4. From Figure 10, navigate to the Users page by clicking on the option that reads *Users* in the left hand blade. The screen should resemble the following below:
```

Comment:
I don't know if numbering the figures is going to scale. You already missed references to figures, like this one which still refers to "Figure 10".

Reply:
The alt text on the figures also refer to the old numbers (e.g., On line 287 the alt text is "figure 10", when it should be "Home page of the Teams Admin portal"

Reply:
@aosolis - please look at the latest commit on the README.md file. This should have all the image references properly adjusted.

Reply:
@coderkrishna The alt text on the image matters, as it is visible to users. We can't have the alt text on Figure 20 be "figure10".

I recommend replacing them with something descriptive (as the alt text is intended to be), so you don't have to worry about keeping it in sync with the figure numbering.